src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -22,576 +22,574 @@ internal class StyleViewModel : AbstractOptionPreviewViewModel
     {
         #region "Preview Text"
 
-        private static readonly string s_fieldDeclarationPreviewTrue = @"
-class C{
-    int capacity;
-    void Method()
-    {
+        private static readonly string s_fieldDeclarationPreviewTrue = $@"
+class C{{
+    int {ServicesVSResources.capacity};
+    void M()
+    {{
 //[
-        this.capacity = 0;
+        this.{ServicesVSResources.capacity} = 0;
 //]
-    }
-}";
+    }}
+}}";
 
-        private static readonly string s_fieldDeclarationPreviewFalse = @"
-class C{
-    int capacity;
-    void Method()
-    {
+        private static readonly string s_fieldDeclarationPreviewFalse = $@"
+class C{{
+    int {ServicesVSResources.capacity};
+    void M()
+    {{
 //[
-        capacity = 0;
+        {ServicesVSResources.capacity} = 0;
 //]
-    }
-}";
+    }}
+}}";
 
-        private static readonly string s_propertyDeclarationPreviewTrue = @"
-class C{
-    public int Id { get; set; }
+        private static readonly string s_propertyDeclarationPreviewTrue = $@"
+class C{{
+    public int {ServicesVSResources.Id} {{ get; set; }}
     void Method()
-    {
```

Comment:
In some places you've kept Method, but elsewhere renamed to M. Is this deliberate and if so, what's the rationale?